src/pages/signin/SignInModal.tsx

```diff
@@ -23,6 +24,13 @@ function SignInModal() {
     // More info https://github.com/Expensify/App/pull/62799#issuecomment-2943136220.
     const SignInPageBase = useMemo(() => (isMobileSafari() ? SignInPageWrapped : SignInPage), []);
 
+    // The SignInPage (child component of SignInModal) uses useHandleBackButton, which adds a hardwareBackPress listener that remains active in the SignInModal.
+    // Use of useHandleBackButton with a returning true callback disables the default SignInModal hardware Android button behaviour, leaving only SignInPage handling (https://github.com/Expensify/App/issues/69391).
```

Comment:
Appreciate the detailed comment. But as we're fixing this could we remove the github URLs and leave the explanations.